components/eamxx/cime_config/testdefs/testmods_dirs/scream/mam4xx/all_mam4xx_procs/shell_commands

```diff
@@ -0,0 +1,22 @@
+#------------------------------------------------------
+# MAM4xx adds additional tracers to the simulation
+# Increase the number of tracers for MAM4xx simulations
+#------------------------------------------------------
+$CIMEROOT/../components/eamxx/cime_config/testdefs/testmods_dirs/scream/mam4xx/update_eamxx_num_tracers.sh
+
+#modify initial condition file to get aerosol species ICs
+$CIMEROOT/../components/eamxx/scripts/atmchange initial_conditions::Filename='$DIN_LOC_ROOT/atm/scream/init/screami_mam4xx_ne4np4L72_c20240208.nc' -b
+
+# Add all MAM4 processes (except ACI)
+$CIMEROOT/../components/eamxx/scripts/atmchange physics::atm_procs_list="mam4_constituent_fluxes, mac_aero_mic,mam4_wetscav, mam4_optics, rrtmgp, mam4_srf_online_emiss,mam4_drydep" -b
+
+# Add mam4_aci in mac_aero_mic
+$CIMEROOT/../components/eamxx/scripts/atmchange mac_aero_mic::atm_procs_list="tms,shoc,cldFraction,mam4_aci,p3" -b
+
+#Set precribed ccn to false so that P3 uses input from ACI
+$CIMEROOT/../components/eamxx/scripts/atmchange p3::do_prescribed_ccn=false -b
+
+#Set predicted ccn to true so that P3 uses input from ACI
+$CIMEROOT/../components/eamxx/scripts/atmchange p3::do_predict_nc=true -b
```

Comment:
We tried to mimic EAMv2 process ordering, so the order matters. We can switch the order (an EAMxx feature), but I am not sure what the implications will be. It can be a nice research topic for a project to study the implications of process ordering. 

mam4_aci is for aci, so it has to be called with P3 and SHOC. That is why it is part of the `mac_aero_mic` process group. The rest of the processes are part of the regular time step. The order is the same as v2 (except for the application of surface fluxes, which mimics v3).